src/ServiceManagerBase.sol

```diff
@@ -3,20 +3,25 @@ pragma solidity =0.8.12;
 
 import {OwnableUpgradeable} from "@openzeppelin-upgrades/contracts/access/OwnableUpgradeable.sol";
 
+import {BitmapUtils} from "src/libraries/BitmapUtils.sol"; 
 import {ISignatureUtils} from "eigenlayer-contracts/src/contracts/interfaces/ISignatureUtils.sol";
 import {IDelegationManager} from "eigenlayer-contracts/src/contracts/interfaces/IDelegationManager.sol";
 
 import {IServiceManager} from "src/interfaces/IServiceManager.sol";
 import {IRegistryCoordinator} from "src/interfaces/IRegistryCoordinator.sol";
+import {IStakeRegistry} from "src/interfaces/IStakeRegistry.sol";
 
 /**
  * @title Minimal implementation of a ServiceManager-type contract.
  * This contract can inherited from or simply used as a point-of-reference.
  * @author Layr Labs, Inc.
  */
 contract ServiceManagerBase is IServiceManager, OwnableUpgradeable {
-    address immutable registryCoordinator;
+    using BitmapUtils for *;
```

Comment:
nit: can we change this

Reply:
updated since we're calling into library directly